### PR TITLE
343808 - Add presence and format validation on Card name

### PR DESCRIPTION
**Pull request by nicolas-brousse**

### CODE REVIEW

File: spec/models/composant_spec.rb [hunk 1]
```diff
@@ -17,6 +17,22 @@
 
   describe "validations" do
     xit { is_expected.to be_valid }
```
Comment (B-Rass) [inline spec/models/composant_spec.rb:19]: Did you look why this test is deactivated?

**Review by B-Rass — CHANGES_REQUESTED** [review]
Maybe we should make it more explicit. Or make the field form red? Or both?

File: spec/models/composant_spec.rb (hunk 1 above)
Comment (nicolas-brousse) [reply spec/models/composant_spec.rb:19]: The object need to have associations set correctly for that

**Comment by nicolas-brousse** [comment]
> Maybe we should make it more explicit. Or make the field form red? Or both?

I improved the error message

**Review by B-Rass — APPROVED** [review]

**Pull request merged**